frontend/apps/reader/modules/readergesture.lua

```diff
@@ -248,7 +248,7 @@ function ReaderGesture:init()
         local reset = false
         for k, v in pairs(mirrored_if_rtl) do
             -- We only replace them if they are still the other direction's default.
-            -- If not, the user has changed them: let him deal with setting new ones if needed.
+            -- If not, the user has changed them: Let it deal with setting new ones if needed.
```

Comment:
```suggestion
            -- If the user has changed directions, they can deal with setting new ones if needed.
```

They is a gender-neutral third person pronoun, but it is not normally used for people.

Reply:
If we need to be that excessively politically correct in comments, I'm gonna stop putting comments :|

Reply:
Ftr, I'm objecting exclusively to "it".

Reply:
Yep, purely grammatical: *it* for a person just doesn't work ;).

Reply:
It isn't political, just poor English. There is no need to personify the user, as it is a concept.
If for example, (without that) "If anyone wants to use the fire escape in case of a fire, they will find it around the corner." "they" can be employed.

Reply:
If we can't use poor english in comments, I'm gonna stop putting comments :|

(But I doubt it's worse than "it" or "they"... I often use "the user" when I refer to users-only in comments - when I write about how developpers would/should do things, I write "you". I respect both of them, and neither is more "a concept" than the others ! They're both human beings :)

And at least, "him", personifying the user, drop some ambiguity:
What would be "it" in your initial suggestion? the user? the code that follows? the widget class we're implementing? the widget object that will be instantiated?
With "him", no ambiguity.

So please, leave this comment as it is - and ignore the above suggestion as it is dropping some contextual logic by removing the leading `If not, `. Unclear logic is worse than poor english.

Reply:
Using "he" is perhaps slightly outdated stylistically, but it's definitely not poor English.

Reply:
@poire-z: I *really* wouldn't worry about that ;). We're being nitpicky *because* the PR itself is nitpicky. Kind of a vicious circle ;).

Reply:
Thanks :) but I don't _really_ worry - if we can't put a bit of drama-dy in our GitHub comments, I'm gonna stop writting GitHub comments! :)
Just trying to discourage rich-english translators from too easily thinking they can change comments or non-translated strings for pseudo grammar reasons, without having to read and understand the code and logic surrounding these comments (because the code _is_ some comment to the comments - in case the comments are not clear enough :) - better they not contradict each others).

Reply:
So you use "he", for archaic reasons, sure. The odds of anyone thinking that isn't in poor taste and for no good reason is slim. Slimmer still considering the state of the rest of the strings in KOReader. Speaking of users, lets have them, and lets consider what they are likely to understand rather than what is merely a technically dependable position.

Reply:
Doesn't change the fact that `it` is ungrammatical in this context?

(TL;DR: While "he" isn't great, at least it makes a proper sentence).